### DISCUSSION TITLE
Enable mps in my macbook

### DISCUSSION THREAD

**Discussion by arthurvaz05** [amazon-science/chronos-forecasting#142] [Q&A] [answered]
Hi everyone,

I'm encountering an error while trying to enable MPS on my MacBook, which has an M3 Max chip. The error message I'm receiving is:

```
 NotImplementedError: The operator 'aten::isin.Tensor_Tensor_out' is not currently implemented for the MPS device. If you want this op to be added in priority during the prototype phase of this feature, please comment on https://github.com/pytorch/pytorch/issues/77764. As a temporary fix, you can set the environment variable `PYTORCH_ENABLE_MPS_FALLBACK=1` to use the CPU as a fallback for this op. WARNING: this will be slower than running natively on MPS.
```

Does anyone have a solution or workaround for this issue? Thanks in advance!

**Comment by lostella** [comment]
Hi, did you try setting the environment variable as suggested be the error message?

**Comment by abdulfatir** [comment, marked as answer]
@arthurvaz05 Please check my response to #141. You need to downgrade your `transformers` version.
